txprocessor/txprocessor.go

```diff
@@ -1038,7 +1043,11 @@ func (tp *TxProcessor) applyTransfer(coordIdxsMap map[common.TokenID]common.Idx,
 						accSender.TokenID, coordIdxsMap[accSender.TokenID]))
 			}
 			// accumulate the fee for the Coord account
-			accumulated := tp.AccumulatedFees[accCoord.Idx]
+			accumulated, ok := tp.AccumulatedFees[accCoord.Idx]
+			if !ok {
+				accumulated = big.NewInt(0)
+				tp.AccumulatedFees[accCoord.Idx] = accumulated
+			}
 			accumulated.Add(accumulated, fee)
```

Comment:
What you think refactor this snippet into a method from `tp` or outside it?

Reply:
https://github.com/hermeznetwork/hermez-node/pull/799#discussion_r640754615